testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -539,13 +537,15 @@ def params_from_base_suite_setup(request):
             if test.rep_call.failed:
                 failed_test_list.append(test.rep_call.nodeid)
         zip_data = suite_cbllog.get_logs_in_zip()
-        suite_log_zip_file = "Suite_test_log_{}.zip".format(str(time.time()))
+        suite_log_zip_file = "Suite_test_log_{}.zip"
```

Comment:
do you intent to replace {} somewhere? didn't find it. if not, the file name doesn't have to contain {}, which causes confusion.